sound/soc/sof/sof-audio.c

```diff
@@ -270,8 +270,8 @@ sof_unprepare_widgets_in_path(struct snd_sof_dev *sdev, struct snd_soc_dapm_widg
 	struct snd_sof_widget *swidget = widget->dobj.private;
 	struct snd_soc_dapm_path *p;
 
-	/* no need to process sink */
-	if (!swidget->prepared)
+	/* no need to process sink. And return if it is used by other pipelines in mixer case */
```

Comment:
@RanderWang it might be better to say "return if the widget is in use or if it is already unprepared"

Reply:
sure， thanks